.rubocop.yml

```diff
@@ -9,10 +9,21 @@ AllCops:
 
 # Cop's to ignore
 
+
+# It thinks that Ruby version dependend definitions are duplicates
+Bundler/DuplicatedGem:
+  Exclude:
+    - "Gemfile"
+
```

Comment:
see https://github.com/bbatsov/rubocop/issues/3752 if interessted in more details